doc/03_research/03_planning/Implementation.md

```diff
@@ -0,0 +1,135 @@
+# Planning Implementation
+
+**Summary:** 
+The document gives a first impression of how the planning could/should work
+and how the topics are edited
+---
+
+## Author
+
+Simon Erlbacher, Niklas Vogel
+
+## Date
+
+29.11.2022
+
+## Prerequisite
+
+
+---
+<!-- TOC -->
+* [Planning Implementation Details](#coding-style-guidelines)
+  * [Author](#author)
+  * [Date](#date)
+  * [Prerequisite](#prerequisite)
+  * [Visualization](#visualization)
+  * [Preplanning](#preplanning)
+  * [Local Path Planning](#local-path-planning)
+  * [Decision Making](#decision-making)
+  * [Next steps](#next-steps)
+* [Sources](#sources)
+<!-- TOC -->
+
+---
+## **Visualization** 
+
+![Implementation](../../00_assets/Planning_Implementierung.png)
+
+---
+## **Preplanning**
+  
+The Preplanning receives the next [target point](./basics.md#wie-sehen-die-daten-vom-leaderboard-fr-das-global-planning-aus) from the Carla Leaderboard. Also it reads the map data which is in [OpenDrive](https://www.asam.net/standards/detail/opendrive/) format. The [commonroad route planner (CRP)](https://gitlab.lrz.de/tum-cps/commonroad-route-planner/) from TUM can be used to calculate a Lanelet model (Note that almost all old PAF-Projects have adapted versions of the CRP). Internally the CRP converts the Map into a Scenario Object. When given a start and goal position and the current orientation of the vehicle a so called planning problem is created by the planer. The Orientation can be calculated on the Odometry and the GNUU data from the sensoring area. The Output (Solution of the planning problem) will be a route defined by a sequence of lanelets and a sequence of points (~ 10cm apart).
+
+Lanelet Model Example :
+![Lanelet Model Example](../../00_assets/Lanelets.png)[[Source]](https://github.com/ll7/psaf2/tree/main/Planning/global_planner)
+
+
+Input: 
```

Comment:
In- und Output stimmten nicht vollständig damit überein, was in PR #56 zu Issue #33 angegeben ist. Muss hier PR #56 angepasst werden?